packages/tasks/src/model-libraries.ts

```diff
@@ -34,7 +34,7 @@ export interface LibraryUiElement {
 	 * By default, those files are counted:
 	 * "config.json", "config.yaml", "hyperparams.yaml", "meta.yaml"
 	 */
-	countDownloads?: ElasticBoolQueryFilter;
+	countDownloads?: ElasticBoolQueryFilter[];
```

Comment:
what is the behavior of the array? is it an `AND`?

Reply:
AND